### PR TITLE
update model funcs

### PR DESCRIPTION
**Pull request by double-y**
### Description
* modify an error message

* fix a test case for conf_int

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [x] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/build_glm.R [hunk 1]
```diff
@@ -88,7 +88,7 @@ build_glm <- function(data, formula, ..., keep.source = TRUE, augment = FALSE, g
     ret
   }, error = function(e){
     if(e$message == "contrasts can be applied only to factors with 2 or more levels"){
-      stop("more than 2 unique values are needed for categorical predictor columns")
+      stop("more than 1 unique value is expected for categorical columns assigned as predictors")
```
Comment (hideaki) [inline R/build_glm.R]: Should be , "more than 1 unique values are expected..."

File: R/build_lm.R [hunk 2]
```diff
@@ -85,7 +85,7 @@ build_lm <- function(data, ..., keep.source = TRUE, augment = FALSE, group_cols
     ret
   }, error = function(e){
     if(e$message == "contrasts can be applied only to factors with 2 or more levels"){
-      stop("more than 2 unique values are needed for categorical predictor columns")
+      stop("more than 1 unique value is expected for categorical columns assigned as predictors")
```
Comment (hideaki) [inline R/build_lm.R]: Should be , "more than 1 unique values are expected..."

File: R/build_glm.R (hunk 1 above)
Comment (double-y) [inline R/build_glm.R]: Right, will modify so

File: R/build_lm.R (hunk 2 above)
Comment (double-y) [inline R/build_lm.R]: Right, will modify so

**Pull request merged**